googleplayreviews.js

```diff
@@ -13,7 +14,7 @@ exports.startReview = function (config) {
             }
 
             appInformation.appName = appData.title;
-            appInformation.appIcon = 'https:' + appData.icon;
+            appInformation.appIcon = appData.icon;
```

Comment:
is `https:` being added somewhere else or is it no longer required?

Reply:
I checked the output and it seems like `appData.icon` includes a URL with `https` already.